### PR TITLE
19449 getting started

### PR DESCRIPTION
**Pull request by HG00**
**What this PR does / why we need it**: Sentence casing all headings in doc

**Which issue(s) this PR fixes**: #19449

### CODE REVIEW

File: docs/sources/auth/generic-oauth.md [hunk 1]
```diff
@@ -206,9 +206,10 @@ allowed_organizations =
     allow_sign_up = true
     client_id = <OpenID Connect Client ID from Centrify>
     client_secret = <your generated OpenID Connect Client Sercret"
-    scopes = openid email name
+    scopes = openid profile email
```
Comment (torkelo) [inline docs/sources/auth/generic-oauth.md]: why this change? this changes the example config of this oauth provider

Comment (HG00) [reply docs/sources/auth/generic-oauth.md]: This change changes the example config so that it actually works :) - We're using Centrify (now Idaptive), and it breaks with the example config. After a lot of messing around, "openid profile email" worked for us. We could also use just email, but this would mean new users did not have name field filled.

Comment (HG00) [reply docs/sources/auth/generic-oauth.md]: PS not that clear, but you can see from the examples that "name" is not an available scope: https://developer.idaptive.com/docs/openid-connect-_new

Comment (torkelo) [reply docs/sources/auth/generic-oauth.md]: ah, good! could have mentioned that in the description :) Got confused as the rest of the changes where of such different type

**Review by torkelo — APPROVED** [review]

File: docs/sources/auth/generic-oauth.md (hunk 1 above)
Comment (HG00) [reply docs/sources/auth/generic-oauth.md]: That would have made a lot more sense, sorry, next time! :)
